### PR TITLE
Links and includes fixes

### PR DESCRIPTION
**Pull request by trodrigues**
- Clarify the default level of inclusions
- Clarify how to get no inclusions
- Fix wrongly documented parameter name on endpoint documentation

### CODE REVIEW

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 1]
```diff
@@ -196,7 +196,9 @@ Learn more in the [developer center](https://www.contentful.com/developers/docs/
 
 ## Retrieval of linked items [/spaces/{space_id}/entries?access_token={access_token}&include={value}]
 
-When you have related content (e.g. entries with links to image assets) it's possible to include both search results and related data in a single request. To do so, the `include` parameter should be used, specifying the number of levels to be resolved. The maximum number of inclusions is `10`.
+When you have related content (e.g. entries with links to image assets) it's possible to include both search results and related data in a single request. To do so, the `include` parameter should be used, specifying the number of levels to be resolved.
+
+The maximum number of inclusions is `10`. The default number if the parameter is not specified, is 1. If you want no inclusions at all, you should set it to 0.
```
Comment (realityking) [inline apiary/_partials/delivery-and-preview-resources.apib]: a) Either all the numbers should be inline blocks or none of them.
b) `The default number if the parameter` missing comma before `if`

File: docs/concepts/links.md [hunk 2]
```diff
@@ -35,6 +35,9 @@ When you have related content (e.g. entries with links to other entries) it's po
 
 Simply tell the search to include the targets of links in the response: Set the `include` parameter to the number of levels you want to resolve. The maximum number of inclusion is 10.
```
Comment (realityking) [inline docs/concepts/links.md]: I know you didn't touch that, but it should be "maximum number of inclusion**s**"

File: docs/concepts/links.md [hunk 3]
```diff
@@ -49,6 +52,9 @@ Link resolution works regardless of how many results are there in `items`. Some
 
 Querying linked items is as simple as adding a specific `include` parameter to retrieve a desired level of related content to be displayed in an application.
 
+{: .note}
+**Note:** When omitted, the `include` parameter takes the standard value of `1`. If you want no links at all to be included, you should set the value to 0.
```
Comment (realityking) [inline docs/concepts/links.md]: Again, either both numbers should be inline code blocks or none of them.

**Comment by realityking** [comment]
Good change but a few style issues.

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 4]
```diff
@@ -207,7 +209,7 @@ Note that only links between entries, spaces and assets are resolved. Links betw
 + Parameters
     + space_id (required, string, `cfexampleapi`) ... Alphanumeric `id` of the space to retrieve.
     + access_token (required, string, `b4c0n73n7fu1`) ... :[token description](tokentype)
-    + value (required, number, `2`) ... Number of levels to include.
+    + include (required, number, `2`) ... Number of levels to include (defaults to 1)
```
Comment (realityking) [inline apiary/_partials/delivery-and-preview-resources.apib]: Sorry, missed that one. This was actually correct, as it's the name of the variable in the URL Template. If you want to change it you also need to change the template. (Line 197)

Kinda shocked the tests don't fail for this 😲

Comment (trodrigues) [inline apiary/_partials/delivery-and-preview-resources.apib]: Fixed

**Pull request merged**